pages/docs/react/latest/migrate-from-v3.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: Migrate from JSX v3
 description: "Migrate from JSX v3"
-canonical: "/docs/react/latest/migrate-react"
+canonical: "/docs/react/latest/migrate-from-v3"
```

Comment:
I am very confused by this change. Current master has a `latest/migrate-react`. So this change is essentially adding a new page `migrate-from-v3` (which I agree is a way better url, although I'd probably call it `migrate-from-jsx-3` instead).

Now that the change is out there, ppl may still refer to it, so we might need a redirection to this page via the nextjs config.

This is quite an unrelated change, so I'd want this to be a separate PR to master.